public/index.html

```diff
@@ -7,14 +7,18 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="play your online role-playing games with your friends without leaving home."
     />
+    <meta name="google-signin-scope" content="profile email">
+    <meta name="google-signin-client_id" content="129422048752-f6c7ajp6j4pk7s9u6gkrcohjpaasq430.apps.googleusercontent.com">
```

Comment:
The GOOGLE_CLIENT_ID is defined in the [GoogleBtn component](https://github.com/garan-no-dou/role-ui/pull/3/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-8e1040cfc10cfa7eac6a5d521baf1e26R6), do we need to define it as well in the index.html?

Reply:
resolved